perf/bench/netflix/prep

```diff
@@ -0,0 +1,40 @@
+#! /bin/sh
+#
+# Fetch rules and targets prior for the "r2c" benchmark.
+#
+# rule_dir: input/rules
+# target_dir: input/lemur
+#
+# Uses sh because bash is not installed within the semgrep docker container.
+#
+set -eu
+
+mkdir -p input
+mkdir -p input/rules
+
+cp r2c-ci.yml input/rules
```

Comment:
if we plan to use that a lot, maybe we could share them instead of duplicating them, so if we update them, we have just to update them at one place.
maybe in perf/common_rules/ @mjambon ?